nautobot/docs/user-guide/administration/configuration/authentication/sso.md

```diff
@@ -42,7 +42,7 @@ Install the system dependencies as `root`:
 sudo apt install -y libxmlsec1-dev libxmlsec1-openssl pkg-config
 ```
 
-Install the `sso` Python extra as the `nautobot` user.
+Install the `sso` Python extra as the `nautobot` user.  It is included in the standard docker image but the system dependencies are not.
```

Comment:
Huh. That may be a bug in and of itself... looking at the Dockerfile, we do install the above packages but only as `dev` dependencies, rather than as part of the `final` image. We may want to change that.

Reply:
I'll remove that bit, then.  Should I open an issue for the incorrectly included packages while I'm at it?

Reply:
Fine to leave in for now if you like, as it's accurate enough at the moment. :-) Feel free to open the other issue. Thanks!

Reply:
Looks like a regression, actually 😞 We fixed this not too long ago in #4282, but the 2.0 release brought in some broader restructuring of the Docker build and it looks like we lost that fix in the merge resolution. Sorry about that!

Reply:
[Issue 5332](https://github.com/nautobot/nautobot/issues/5332)